Podfile

```diff
@@ -6,14 +6,14 @@ project 'Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload.xcodeproj'
 target 'VimeoUpload' do
     platform :ios, '8.0'
     use_frameworks!
-    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'develop'
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'feature/total-storage'
```

Comment:
Should this be pointing to develop of VimeoNetworking? For here and other 2 targets